update Software kibana instructions to account for sidecar-logging (#3672)
astronomer/docs: software/kibana-logging.md

2. Click **Menu** in the upper left, click **Stack Management**, and then click **Index Patterns**.

> 3. In the **Name** field enter `fluentd.*` (or if using sidecar logging `vector.*`) and then click **Create index pattern**.

Reviewer: ```suggestion
3. In the **Name** field enter `fluentd.*` or if using sidecar logging, enter `vector.*`, and then click **Create index pattern**.
```

> 3. In the **Name** field enter `fluentd.*` or if using sidecar logging, enter `vector.*`, and then click **Create index pattern**.

Reviewer: ```suggestion
3. In the **Name** field enter `fluentd.*` or enter `vector.*` if using sidecar logging, and then click **Create index pattern**.
```

>     Elasticsearch uses [index patterns](https://www.elastic.co/guide/en/kibana/current/index-patterns.html) to organize how you explore data. Setting `fluentd.*` (or if using sidecar logging, `vector.*`) as the index means that Kibana will display all logs from all deployments.

Reviewer: ```suggestion
    Elasticsearch uses [index patterns](https://www.elastic.co/guide/en/kibana/current/index-patterns.html) to organize how you explore data. Setting `fluentd.*`, or `vector.*` if using sidecar logging,  as the index means that Kibana displays all logs from all deployments.
```
